nbs/15_migrate.ipynb

```diff
@@ -28,7 +28,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def _cat_slug(d):
</pre><p>In the spirit of being concise, I'd personally prefer <code>d</code> as the param name over <code>fmdict</code> in the other notebooks too </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>